Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx
```

Comment:
With this dockerfile ,
it will be a two step process.
First building manually and then verifying in the container if things are good.
The DockerFile which we made yesterday seems better to me .
Also we may have a separate dockerfile for dev env to ensure all the env stages including dev are within containers.
Thoughts ??

Reply:
That sounds good.
However, the build process is supposed to be part of a CI job. 
Building within the Dockerfile seems counterintuitive.

This also solves the problem of having to deal with environments while building the docker image. These are things our CI pipeline will handle.

Reply:
I would agree to shubham's point here.
IMHO It more of circleCI's job to build rather than doing it in docker.